src/main/java/org/jabref/gui/fieldeditors/FieldEditors.java

```diff
@@ -52,10 +51,10 @@ public static FieldEditorFX getForField(final Field field,
                 preferences.getBoolean(JabRefPreferences.ENFORCE_LEGAL_BIBTEX_KEY),
                 preferences.getBoolean(JabRefPreferences.ALLOW_INTEGER_EDITION_BIBTEX));
 
-        final boolean isSingleLine = FieldFactory.isSingleLineField(field);
+         boolean isMultiLine = FieldFactory.isMultiLineField(field, preferences.getFieldContentParserPreferences().getNonWrappableFields());
```

Comment:
Can we use an `enum` here? I really don't like the booleans - and we see changing the flag from `singleLine` to `multiLine` was necessary of readability. (Otherweise, the variable would be kept to flag `singleLine` - and negation would be used)

I propose to use a boolean enum as proposed at https://stackoverflow.com/a/12713550/873282

```java
enum USED_LINES {
    MULTI(true), SINGLE(false);
...
```

Maybe, just a simple enum would also be OK. Then, the construction from boolean would be a bit more tricky though.

Reply:
If you want to go in this direction, an enum `FieldType` with `Text` and `TextLong` would make the most sense and conveys the semantics. Can then be extended in the future to include `Keywords`, `Numeric` etc.

Reply:
I really would like to leave it for the moment and implement a more generic solution in a follow up PR. I think about integrating it in the custom entry types dialog.